ULC/README.md

```diff
@@ -64,6 +64,27 @@ Saved: y
 ```
 Note: Consequently `let` and `=` are keywords, and so you cannot name variables with these.
 
+You can read and run a file of commands by prefixing the filepath with `f`. This will behave as if each non-empty line of the file were entered into the REPL manually. Terms previously added to the environment via `let` commands will be in scope for later commands, so this can be used for building a library of term definitions. For example, if the file `bool.ulc` contains the following:
+```
```

Comment:
It's worth adding this as an actual file, or more generally a larger prelude file with lots more definitions for e.g. lists, bools, Y, etc...